scripts/install

```diff
@@ -101,10 +101,18 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 #=================================================
 ynh_script_progression --message="Configuring Kanboard..."
 
+# Retrieve admin email
+email=$(ynh_user_get_info --username=$admin --key=mail)
+
 # Copy and edit config.php
-config_php="$final_path/config.php"
+config_php="${final_path}/config.php"
 
-ynh_add_config --template="../conf/config.php" --destination="$config_php"
+cp ../conf/config.php "$config_php"
+ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file="$config_php"
+ynh_replace_string --match_string="__DB_NAME__" --replace_string=$db_name --target_file="$config_php"
+ynh_replace_string --match_string="__USER__"    --replace_string=$admin   --target_file="$config_php"
+ynh_replace_string --match_string="__EMAIL__"   --replace_string=$email   --target_file="$config_php"
+ynh_replace_string --match_string="__DOMAIN__"  --replace_string=$domain  --target_file="$config_php"
```

Comment:
Eeeh, why revert this ?

Reply:
I reverted because of this variable `__DIR__` that looks like a variable to be replaced, with the same format of what ynh_add_config is looking for... But it is defined by the application and should not be replaced. 
https://github.com/YunoHost-Apps/kanboard_ynh/blob/9e979b0d2f450971e38adf8ebad2c3ea9a3ce8cd/conf/config.php#L4